mibidata/segmentation.py

```diff
@@ -3,13 +3,14 @@
 Copyright (C) 2019 Ionpath, Inc.  All rights reserved."""
 
 import numpy as np
+from numpy.testing import assert_array_equal
 import pandas as pd
 from scipy import ndimage as ndi
 
-from mibidata import mibi_image as mi
+from mibidata import mibi_image as mi, util
 
 
-def extract_cell_dataframe(label_image, image=None, mode='total'):
+def extract_cell_dataframe(label_image, image=None, mode='total', n_sec=8):
```

Comment:
Descriptive rather than abbreviated parameter names are generally prefered, e.g. `sectors` or even `num_sectors` rather than `n_sec`. Final decision up to you though.

Reply:
done